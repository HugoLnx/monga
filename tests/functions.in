void func(int num, int num2) {
	printf("num: %d\n", num);
	{printf("num2: %d\n", num2);}
}

void main() {
	func(15, 20);
	func(25, 30);
}
