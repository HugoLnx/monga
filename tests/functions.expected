num: 15
num2: 20
num: 25
num2: 30
func2(90): 190
func2(95): 195
